moveit_fake_controller_manager/src/moveit_fake_controller_manager.cpp

```diff
@@ -324,15 +260,19 @@ class MoveItFakeControllerManager : public moveit_controller_manager::MoveItCont
   }
 
   /* Cannot switch our controllers */
-  virtual bool switchControllers(const std::vector<std::string> &activate, const std::vector<std::string> &deactivate) { return false; }
+  virtual bool switchControllers(const std::vector<std::string> &activate, const std::vector<std::string> &deactivate)
+  {
+    return false;
+  }
 
 protected:
-
   ros::NodeHandle node_handle_;
-  std::map<std::string, moveit_controller_manager::MoveItControllerHandlePtr> controllers_;
+  ros::Publisher pub_;
+  std::map<std::string, BaseFakeControllerPtr> controllers_;
 };
 
-} // end namespace moveit_fake_controller_manager
+}  // end namespace moveit_fake_controller_manager
 
+#include <pluginlib/class_list_macros.h>
```

Comment:
Does need to. As this is only required for declaration of the plugin, I prefer to have them together.
This way, you cannot forget to (re)move the include if the plugin declaration is (re)moved.